arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,393 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/replicationLinks/{linkId}": {
+      "delete": {
+        "tags": [
+          "DatabaseReplicationLinks"
+        ],
+        "operationId": "Databases_DeleteReplicationLink",
+        "description": "Deletes the Azure SQL Database Replication Link with the given id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure SQL Database that has the replication link to be dropped."
+          },
+          {
+            "name": "linkId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the replication link to be deleted."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "OK"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DatabaseReplicationLinks"
+        ],
+        "operationId": "Databases_GetReplicationLink",
+        "description": "Returns information about an Azure SQL Database Replication Link.",
```

Comment:
#Resolved

---
In reply to: [85474934](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85474934) [](ancestors = 85474934)